nn/init.go

```diff
@@ -420,3 +420,35 @@ func XavierUniform_(x *ts.Tensor, gainOpt ...float64) {
 
 	src.MustDrop()
 }
+
+// XavierNormal fills the input tensor with values according to the method
+// described in the paper `Understanding the difficulty of training deep feedforward neural networks`
+// using a normal distribution
+//
+// Also known as normal Glorot initialization.
+//
+// Paper: https://proceedings.mlr.press/v9/glorot10a/glorot10a.pdf
+// Pytorch implementation: https://github.com/pytorch/pytorch/blob/df50f91571891ec3f87977a2bdd4a2b609d70afc/torch/nn/init.py#L337
+func XavierNormal_(x *ts.Tensor, gainOpt ...float64) {
```

Comment:
- We try to use Go conventional naming. If it's a unexposed function we can start with lowercase `xavierNormal()`
- There a placeholder for GolorotInit at https://github.com/sugarme/gotch/blob/cd0565716fd369786b0fe016e38c46ba4046bf37/nn/init.go#L300 . It would be great if you could fill in those things. 
- Try some unit test or example (make sure accuracy and no memory leak are the most priority as the function will be used everywhere (I often test result against Python in terms of accuracy).